incubator/binding-risingwave.spec/src/main/scripts/io/aklivity/zilla/specs/binding/risingwave/streams/transformation/create.materialized.view/client.rpt

```diff
@@ -0,0 +1,185 @@
+
+read advised zilla:flush ${pgsql:flushEx()
+                              .typeId(zilla:id("pgsql"))
+                              .completion()
+                                .tag("CREATE_MATERIALIZED_VIEW") #TODO: double check
+                                .build()
+                              .build()}
+
+read advised zilla:flush ${pgsql:flushEx()
+                              .typeId(zilla:id("pgsql"))
+                              .ready()
+                                .status("IDLE")
+                                .build()
+                              .build()}
+
+write zilla:data.ext ${pgsql:dataEx()
+                              .typeId(zilla:id("pgsql"))
+                              .query()
+                                .build()
+                              .build()}
+write "DESCRIBE MATERIALIZED VIEW active_users;"
+      [0x00]
+
+read advised zilla:flush ${pgsql:flushEx()
+                              .typeId(zilla:id("pgsql"))
+                              .type()
+                                .column()
+                                    .name("balance")
+                                    .tableOid(0)
+                                    .index(0)
+                                    .typeOid(701)
+                                    .length(8)
+                                    .modifier(-1)
+                                    .format("TEXT")
+                                    .build()
+                                .column()
+                                    .name("status")
+                                    .tableOid(0)
+                                    .index(0)
+                                    .typeOid(20)
+                                    .length(8)
+                                    .modifier(-1)
+                                    .format("TEXT")
+                                    .build()
+                                .column()
+                                    .name("user_id")
+                                    .tableOid(0)
+                                    .index(0)
+                                    .typeOid(17)
+                                    .length(-1)
+                                    .modifier(-1)
+                                    .format("TEXT")
+                                    .build()
+                                .build()
+                              .build()}
+
+read advised zilla:flush ${pgsql:flushEx()
+                              .typeId(zilla:id("pgsql"))
+                              .completion()
+                                .tag("DESCRIBE")
+                                .build()
+                              .build()}
+
+read advised zilla:flush ${pgsql:flushEx()
+                              .typeId(zilla:id("pgsql"))
+                              .ready()
+                                .status("IDLE")
+                                .build()
+                              .build()}
+
+
```

Comment:
We might need a barrier here to document that `CREATE TOPIC` will complete before we continue, since topic name is referenced in the `CREATE SINK` command coming next.